projects/eslint-plugin-graphql/tests/parser-test.js

```diff
@@ -50,6 +51,55 @@ describe('parser', function() {
   it('produces reasonable output', function() {
     // we will rely on other more unit tests, or integration tests to ensure the actual complexity is tests
     const result = parser.parseForESLint('fragment apple on Fruit { id }', { filePath: 'some-file.graphql' });
-    expect(result).to.have.keys(['ast', 'scopeManager', 'visitorKeys']);
+    expect(result).to.have.keys(['ast', 'scopeManager', 'visitorKeys', 'services']);
+  });
+
+  describe('parser services', function() {
+    it('has a functioning parse method', function() {
+      const source = `
+query {
+  id
+}
```

Comment:
https://github.com/zspecza/common-tags#stripindenttransformertypeinitial

Reply:
cool library, next time i'll use that. Today switching to that, results in a decent number of test changes due to row/column changes. I'm not sure it's worth it right now, i have though tried to make them all consistent